bin/tag_release

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -evuo pipefail
+go get -u -v github.com/screwdriver-cd/gitversion
+GIT_TAG=$(gitversion --prefix=v bump auto 2>&1|tail -1)
+git push https://$GITHUBKEY@github.com/fiaas/ais $GIT_TAG
```

Comment:
Should GITHUBKEY be added to .travis.yml?

Reply:
We can do that sure. I'll add it.